testpmd/hooks/roles/sriov-networks/tasks/main.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: List all workers
+  shell: >
+    {{ oc_tool_path }} get nodes --no-headers=true --selector='!node-role.kubernetes.io/master'
```

Comment:
If you want to list all the workers, you should remove the selector

Reply:
The intention is to only list the workers, not all nodes. I wonder how we ended up detecting 11 workers

Reply:
upd. checked nodes.txt, we really had 11 workers on cluster1: https://www.distributed-ci.io/jobs/5fa4a100-6c35-4c59-9fa7-a4a641f00740/files 

```
nodes.txt

NAME        STATUS     ROLES    AGE     VERSION            INTERNAL-IP     EXTERNAL-IP   OS-IMAGE                                                        KERNEL-VERSION                 CONTAINER-RUNTIME
master-0    Ready      master   3h9m    v1.23.12+8a6bfe4   192.168.12.11   <none>        Red Hat Enterprise Linux CoreOS 410.84.202302090253-0 (Ootpa)   4.18.0-305.76.1.el8_4.x86_64   cri-o://1.23.5-5.rhaos4.10.gitd9dec98.el8
master-1    Ready      master   3h10m   v1.23.12+8a6bfe4   192.168.12.12   <none>        Red Hat Enterprise Linux CoreOS 410.84.202302090253-0 (Ootpa)   4.18.0-305.76.1.el8_4.x86_64   cri-o://1.23.5-5.rhaos4.10.gitd9dec98.el8
master-2    Ready      master   3h9m    v1.23.12+8a6bfe4   192.168.12.13   <none>        Red Hat Enterprise Linux CoreOS 410.84.202302090253-0 (Ootpa)   4.18.0-305.76.1.el8_4.x86_64   cri-o://1.23.5-5.rhaos4.10.gitd9dec98.el8

worker-0    Ready      worker   158m    v1.23.12+8a6bfe4   192.168.12.21   <none>        Red Hat Enterprise Linux CoreOS 410.84.202302090253-0 (Ootpa)   4.18.0-305.76.1.el8_4.x86_64   cri-o://1.23.5-5.rhaos4.10.gitd9dec98.el8
worker-1    Ready      worker   156m    v1.23.12+8a6bfe4   192.168.12.22   <none>        Red Hat Enterprise Linux CoreOS 410.84.202302090253-0 (Ootpa)   4.18.0-305.76.1.el8_4.x86_64   cri-o://1.23.5-5.rhaos4.10.gitd9dec98.el8
worker-10   NotReady   <none>   146m                       <none>          <none>        <unknown>                                                       <unknown>                      <unknown>
worker-2    Ready      worker   157m    v1.23.12+8a6bfe4   192.168.12.23   <none>        Red Hat Enterprise Linux CoreOS 410.84.202302090253-0 (Ootpa)   4.18.0-305.76.1.el8_4.x86_64   cri-o://1.23.5-5.rhaos4.10.gitd9dec98.el8
worker-3    Ready      worker   157m    v1.23.12+8a6bfe4   192.168.12.24   <none>        Red Hat Enterprise Linux CoreOS 410.84.202302090253-0 (Ootpa)   4.18.0-305.76.1.el8_4.x86_64   cri-o://1.23.5-5.rhaos4.10.gitd9dec98.el8
worker-4    NotReady   <none>   146m                       <none>          <none>        <unknown>                                                       <unknown>                      <unknown>
worker-5    NotReady   <none>   146m                       <none>          <none>        <unknown>                                                       <unknown>                      <unknown>
worker-6    NotReady   <none>   146m                       <none>          <none>        <unknown>                                                       <unknown>                      <unknown>
worker-7    NotReady   <none>   146m                       <none>          <none>        <unknown>                                                       <unknown>                      <unknown>
worker-8    NotReady   <none>   146m                       <none>          <none>        <unknown>                                                       <unknown>                      <unknown>
worker-9    NotReady   <none>   146m                       <none>          <none>        <unknown> 
```

There are some side effects from the sliding migration on the cluster 1. cc: @tonyskapunk

Reply:
I'll close this PR as we do not really need additional logs here and assign the cilab to Tony.